### DISCUSSION TITLE
deleteUser method in firebase_auth

### DISCUSSION THREAD

**Discussion by dzliergaard** [firebase/flutterfire#8472] [Feature request] [answered]
Apple is [requiring](https://transcend.io/blog/apple-requirement-app-account-deletion/) apps with user account creation to provide and surface the ability to delete said accounts. 

['deleteUser'](https://firebase.google.com/docs/auth/web/manage-users#delete_a_user) is not yet supported in firebase_auth, but we need it in order to fulfill this requirement.

**Comment by Ehesp** [comment, marked as answer]
It's supported: https://pub.dev/documentation/firebase_auth/latest/firebase_auth/User/delete.html
